app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    source_file, dest_file = parts[1], parts[2]
```

Comment:
Unpack the parts without indexing and preserving the command name
```suggestion
    command_name, source_file, dest_file, *_ = parts
```